hw/ip/prim/dv/prim_present/prim_present_sim_cfg.hjson

```diff
@@ -45,6 +45,7 @@
     {
       name: smoke
       tests: ["prim_present_smoke"]
+      build_opts: ["+define+NUM_ITER=1"]
```

Comment:
Avoid build opts like its the plague! It creates an unnecessary maintenance overhead. You need a non-smoke version of the test that runs 5000 iterations. If you add that, the build_opt above will end up getting applied to both tests, creating a debug overhead. `build_opts` should NEVER be set in the test specification directly (I thought I added a check in dvsim to flag that as a fatal error). If the use of build_opts is unavoidable, then build opts need to be wrapped into a build_mode. The test then sets the build_mode to use that separate build (so that the rest of the tests can then use the default build mode).  

Please make this a runtime control `run_opts: ["+smoke_test=1"]` to make it consistent with what is done in #4933 and eventually in #4947.

Reply:
will fix!
I'd initially made this a build_opt to match the intent behind the other parameters in the tesbenches, but you're right this particular arg should probably be defined at runtime.